content/en/account_management/audit_trail/events.md

```diff
@@ -357,6 +357,12 @@ See the [Audit Trail documentation][2] for more information on setting up and co
 | [Modify a schedule override][149]  | A user modified a schedule override.  | `@evt.name:"On-Call" @asset.type:override @action:modified`          |
 | [Delete a schedule override][150]  | A user deleted a schedule override.   | `@evt.name:"On-Call" @asset.type:override @action:deleted`           |
 
+### Infrastructure Monitoring
```

Comment:
Don't we need to provide a link to this section on the top of the page under `Overview > Product-Specific Events`?